### PR TITLE
Removes mentions of brig phys request terminal, replaces it with Detective, Give Detective disposals shipping

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do
This fixes Detective's console being unknown and adds it to all 3 maps. It also removes the Brig Phys request console, since it is no longer used.

## Why It's Good For The Game
Brig Phys is gone, Detective's request terminal is broken

## Images of changes
image
image
image
Delta already had a disposal bin

## Changelog
:cl:
fix: Detective's request console is no longer unknown
fix: You can now relay info and request assistance to the Detective's request console
tweak: Detective's office now has a disposal bin, you are also able to ship stuff to it!
/:cl:

### CODE REVIEW

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -509,7 +509,9 @@
 	},
 /obj/machinery/requests_console{
 	name = "Detective Requests Console";
-	pixel_y = 30
+	pixel_y = 30;
+	department = "Detective";
+	departmentType = 5
```
Comment (Miraviel) [inline _maps/map_files/MetaStation/MetaStation.dmm:514]: Request Consoles are able to print Shipping packages, however, the Detective's Office has no disposal bin in their office on MetaStation. Not sure if they need any, but it is a food for thought. If you want to make a full use of the request console, they will need one.

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 2]
```diff
@@ -13336,7 +13336,9 @@
 	},
 /obj/machinery/requests_console{
 	name = "Detective Requests Console";
-	pixel_y = -30
+	pixel_y = -30;
+	department = "Detective";
+	departmentType = 5
```
Comment (Miraviel) [inline _maps/map_files/cyberiad/cyberiad.dmm:13341]: Same as above, if you consider giving the detective a disposal bin, the Cyberiad doesn't have one either.

**Review by Miraviel — CHANGES_REQUESTED** [review]
Tested it on the Cyberiad, it works properly:

<details>
  <summary>Video/screenshots</summary>
  
  

https://user-images.githubusercontent.com/33333517/180660032-1e14ee3d-9991-41c0-82c9-b2de95cafaed.mp4

detective_cyberiad2

detective_cyberiad3

  
</details>

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 1 above)
Comment (Contrabang) [reply _maps/map_files/MetaStation/MetaStation.dmm:514]: Sounds good, I'll change that up and test it in the test commit

**Comment by Contrabang** [comment]
image
The detective can now be properly shipped things! Works on all 3 current maps.

**Review by Miraviel — APPROVED** [review]

**Review by S34NW — CHANGES_REQUESTED** [review]
This is fine and good to go, BUT needs a master update pre-merge to ensure no stealth conflicts on the map files.

**Comment by Contrabang** [comment]
Updated with master, appears everything is working properly

**Review by Sirryan2002 — APPROVED** [review]
code is fine, map changes are fine. Just see S43N's comment before merging

**Comment by AffectedArc07** [comment]
Please merge master branch on this PR to ensure it has no conflicts with the just merged medsci remap

**Comment by Contrabang** [comment]
Updated, tested on box and works fine. No changes to meta or delta so those should work fine too.

**Review by S34NW — APPROVED** [review]

File: code/game/machinery/requests_console.dm [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
 //Radio list
 #define ENGI_ROLES list("Atmospherics","Engineering","Chief Engineer's Desk","Telecoms Admin")
-#define SEC_ROLES list("Warden","Security","Brig Medbay","Head of Security's Desk")
+#define SEC_ROLES list("Warden","Security","Detective","Head of Security's Desk")
 #define MISC_ROLES list("Bar","Chapel","Kitchen","Hydroponics","Janitorial")
 #define MED_ROLES list("Virology","Chief Medical Officer's Desk","Medbay")
 #define COM_ROLES list("Blueshield","NT Representative","Head of Personnel's Desk","Captain's Desk","Bridge")
```
Comment (AffectedArc07) [inline code/game/machinery/requests_console.dm:27]:
```suggestion
#define MISC_ROLES list("Bar", "Chapel", "Kitchen", "Hydroponics", "Janitorial")
#define MED_ROLES list("Virology", "Chief Medical Officer's Desk", "Medbay")
#define COM_ROLES list("Blueshield", "NT Representative", "Head of Personnel's Desk", "Captain's Desk", "Bridge")
```

File: code/game/machinery/requests_console.dm [hunk 4]
```diff
@@ -21,7 +21,7 @@
 
 //Radio list
 #define ENGI_ROLES list("Atmospherics","Engineering","Chief Engineer's Desk","Telecoms Admin")
-#define SEC_ROLES list("Warden","Security","Brig Medbay","Head of Security's Desk")
+#define SEC_ROLES list("Warden","Security","Detective","Head of Security's Desk")
```
Comment (AffectedArc07) [inline code/game/machinery/requests_console.dm:24]:
```suggestion
#define SEC_ROLES list("Warden", "Security", "Detective", "Head of Security's Desk")
```

File: code/game/machinery/requests_console.dm [hunk 5]
```diff
@@ -21,7 +21,7 @@
 
 //Radio list
 #define ENGI_ROLES list("Atmospherics","Engineering","Chief Engineer's Desk","Telecoms Admin")
```
Comment (AffectedArc07) [inline code/game/machinery/requests_console.dm:23]:
```suggestion
#define ENGI_ROLES list("Atmospherics", "Engineering", "Chief Engineer's Desk")
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Formatting + Another redundant one

**Comment by Contrabang** [comment]
Apologies for webedit, currently not available to use VSCode for the forseeable future but dont want to hold up map queue either

File: code/game/machinery/requests_console.dm [hunk 6]
```diff
@@ -20,11 +20,11 @@
...
-#define SEC_ROLES list("Warden","Security","Brig Medbay","Head of Security's Desk")
-#define MISC_ROLES list("Bar","Chapel","Kitchen","Hydroponics","Janitorial")
-#define MED_ROLES list("Virology","Chief Medical Officer's Desk","Medbay")
-#define COM_ROLES list("Blueshield","NT Representative","Head of Personnel's Desk","Captain's Desk","Bridge")
+#define ENGI_ROLES list("Atmospherics", "Engineering", "Chief Engineer's Desk")
+#define SEC_ROLES list("Warden", "Security", "Detective", "Head of Security's Desk")
+#define MISC_ROLES list("Bar", "Chapel", "Kitchen", "Hydroponics", "Janitorial")
+#define MED_ROLES list("Virology", "Chief Medical Officer's Desk", "Medbay")
+#define COM_ROLES list("Blueshield", "NT Representative", "Head of Personnel's Desk", "Captain's Desk", "Bridge")
 #define SCI_ROLES list("Robotics","Science","Research Director's Desk")
```
Comment (AffectedArc07) [inline code/game/machinery/requests_console.dm:28]:
```suggestion
#define SCI_ROLES list("Robotics", "Science", "Research Director's Desk")
```
Forgot this one, my bad

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**